projects/artemis/index.js

```diff
@@ -4,12 +4,47 @@ const { masterchefExports, } = require("../helper/unknownTokens")
 const token = ADDRESSES.harmony.MIS;
 const masterchef = "0x59c777cd749b307be910f15c54a3116ff88f9706";
 
-module.exports = masterchefExports({
-  chain: 'harmony',
-  useDefaultCoreAssets: true,
-  masterchef,
-  nativeToken: token,
-  blacklistedTokens: [
-    '0xed0b4b0f0e2c17646682fc98ace09feb99af3ade', // RVRS bad pricing
-  ]
-})
\ No newline at end of file
+const sdk = require('@defillama/sdk');
+async function goatTvl({block, chain}) {
+  const {WBTC, DOGEB, BTCB} = ADDRESSES.goat;
+  const tokens = ['0xeFEfeFEfeFeFEFEFEfefeFeFefEfEfEfeFEFEFEf', DOGEB, BTCB];
```

Comment:
what is `0xeFEfeFEfeFeFEFEFEfefeFeFefEfEfEfeFEFEFEf`? Is DOGEB pegged to DOGE? We have price support for the BTC assets

Reply:
please use `0x0000000000000000000000000000000000000000` for gas token

Reply:
`0xeFEfeFEfeFeFEFEFEfefeFeFefEfEfEfeFEFEFEf` is only used for reading Artemis contract, the final balance token key is replace with WBTC. 
Shall I use `0x0000000000000000000000000000000000000000` in the final balance object instead of WBTC?

Reply:
WBTC should work too